packages/in_app_purchase/in_app_purchase_storekit/lib/src/store_kit_wrappers/sk_payment_queue_wrapper.dart

```diff
@@ -45,25 +57,21 @@ class SKPaymentQueueWrapper {
   ///
   /// Returns `null` if the user's device is below iOS 13.0 or macOS 10.15.
   Future<SKStorefrontWrapper?> storefront() async {
-    final Map<String, dynamic>? storefrontMap = await channel
-        .invokeMapMethod<String, dynamic>('-[SKPaymentQueue storefront]');
-    if (storefrontMap == null) {
-      return null;
-    }
-    return SKStorefrontWrapper.fromJson(storefrontMap);
+    return SKStorefrontWrapper.convertFromPigeon(await _hostApi.storefront());
   }
 
   /// Calls [`-[SKPaymentQueue transactions]`](https://developer.apple.com/documentation/storekit/skpaymentqueue/1506026-transactions?language=objc).
   Future<List<SKPaymentTransactionWrapper>> transactions() async {
-    return _getTransactionList((await channel
-        .invokeListMethod<dynamic>('-[SKPaymentQueue transactions]'))!);
+    final List<SKPaymentTransactionMessage?> pigeonMsgs =
```

Comment:
You force nonnull down below.  I forget where we ended up with pigeon, can you make this type be nonnull in pigeon?

Reply:
I have no idea how actually, in messages.dart I declare the return type to be List<SKPaymentTransactionMessage>, but it gets generated with the return type of List<SKPaymentTransactionMessage?> 
@bparrishMines could you help me out?

Reply:
Yea, I see that now in the `messages.dart`. If this is what it generates I wouldn't worry about it.  Maybe the later version of pigeon will give you something else.